docs/dev/cli/saucectl/usage/ci/jenkins.md

```diff
@@ -12,15 +12,16 @@ These examples can apply to virtually any Jenkins deployment, provided that you
 
 ## What You'll Need
 
+* A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up)).
```

Comment:
Should all list item end with a period?

Reply:
Same comment as https://github.com/saucelabs/sauce-docs/pull/1100#discussion_r825599689